app/controllers/partners_controller.rb

```diff
@@ -126,12 +126,14 @@ def recertify_partner
 
   def deactivate
     @partner = current_organization.partners.find(params[:id])
-    response = DiaperPartnerClient.put(partner_id: @partner.id, status: "deactivated")
 
-    if response.is_a?(Net::HTTPSuccess) && @partner.update(status: "deactivated")
+    svc = PartnerDeactivateService.new(partner: @partner, organization: current_organization)
```

Comment:
```suggestion
    svc = PartnerDeactivateService.new(partner: @partner)
```
I don't think the organization is necessary as a parameter here?

Reply:
Ah, yeah. I think I needed it in a prev version but forgot to remove. Thanks!